src/cli/index.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+import React from "react";
```

Comment:
```suggestion

import React from "react";
```
Blank line between usr/bin instruction and ts import?

Reply:
Nice catch!